src/Cardano/Wallet.hs

```diff
@@ -26,7 +26,7 @@
 module Cardano.Wallet
     (
     -- * Wallet
-      Wallet
+      Wallet(..)
```

Comment:
Nope! That is, forbidden! 
This would be violating all the guarantees this module provides us.

Reply:
reverted the change